src/logging/NewRelicLoggingService.js

```diff
@@ -45,6 +73,31 @@ function fixErrorLength(error) {
  * @memberof module:Logging
  */
 export default class NewRelicLoggingService {
+  constructor(options) {
+    this.config = options ? options.config : undefined;
+    /*
+        Array of explicit error message regexes. If an error message matches a regex, the error is considered
+        an *ignored* error and submitted to New Relic as a page action - not an error.
+
+        Ignored error regexes are configured per frontend application (MFE).
+
+        For edx.org, add new error message regexes in edx-internal YAML as needed, ordered roughly by
```

Comment:
Non-blocking things to consider:
1. I still think the "For edx.org..." sentence should be moved to the bottom of the docstring. The second sentence here might be ignored outside of edx.org, but it is meant for everyone.
2. Is "ordered roughly by occurrence frequency" due to potential performance concerns? I don't think I'd worry about it, and if you are concerned, couldn't all the regex's be combined into a single regex?

Reply:
Sorry if the answer is obvious, but why did we use the special delim and not just have a single regex? Is it because the regex might get too complicated if a specific error also needed to use `|`? For example, it could have just been:
```
>>> x = re.compile(r"(^\[frontend-auth\] Axios Error|Loading chunk [\d]+ failed. \(error: https://learning.edx.org/[\d\w.]+\.js\))")
>>> x.search('hello')
>>> x.search('[frontend-auth] Axios Error (Request): Network Error (possible local connectivity issue)')
<re.Match object; span=(0, 27), match='[frontend-auth] Axios Error'>
>>> x.search('Loading chunk 8 failed. (error: https://learning.edx.org/8.00acf25bc2384f18cf76.js)')
<re.Match object; span=(0, 83), match='Loading chunk 8 failed. (error: https://learning.>
```
Note: There were some minor issues with your example regex, like some missing or unwanted `\` characters.

Reply:
There's actually no reason why a single regex can't be used here. It would simplify the code in places, which is another plus. The regex itself won't seem any more complex than it currently is - given the `dotenv` restriction of a single string per config env var. I'm going to make this change throughout my three PRs - thanks!

The example regex does need updating - agreed. Here's the example in JS - not Python with its fancy `r''` operator 😸 :
```
> x = new RegExp('^\\[frontend-auth\\] Axios Error|Loading chunk [\\d]+ failed. \\(error: https://learning.edx.org/[\\d\\w\\.]+.js\\)');
/^\[frontend-auth\] Axios Error|Loading chunk [\d]+ failed. \(error: https:\/\/learning.edx.org\/[\d\w\.]+.js\)/
> 'Loading chunk 8 failed. (error: https://learning.edx.org/8.00acf25bc2384f18cf76.js)'.match(x);
[
  'Loading chunk 8 failed. (error: https://learning.edx.org/8.00acf25bc2384f18cf76.js)',
  index: 0,
  input: 'Loading chunk 8 failed. (error: https://learning.edx.org/8.00acf25bc2384f18cf76.js)',
  groups: undefined
]
> 'Loading chunk 4 failed. (error: https://learning.edx.org/4.ebd72a4f92c5e7a0214b.js)'.match(x);
[
  'Loading chunk 4 failed. (error: https://learning.edx.org/4.ebd72a4f92c5e7a0214b.js)',
  index: 0,
  input: 'Loading chunk 4 failed. (error: https://learning.edx.org/4.ebd72a4f92c5e7a0214b.js)',
  groups: undefined
]
> '[frontend-auth] Axios Error (Request): Network Error (possible local connectivity issue) post https://courses.edx.org/login_refresh'.match(x);
[
  '[frontend-auth] Axios Error',
  index: 0,
  input: '[frontend-auth] Axios Error (Request): Network Error (possible local connectivity issue) post https://courses.edx.org/login_refresh',
  groups: undefined
]
```
I attempted a `string.replace()` regex to avoid the user having to escape characters themselves - but I couldn't find one that worked in all cases. For example, is the `.` meant to be a literal `.` - or to match any character? I decided it's best to leave those decisions in the hands of the regex creator.

Reply:
I agree that you don't want to determine where there should be a `\` and where there shouldn't be. However, your example and the actual setting should  not need a `\\` to represent a `\`, right? Or, does it need that to get through the config code?

Reply:
It's apparently got to do with the way JS does string escaping - but I'm not entirely sure why.

My latest commit adds actual JS code which a developer can run to test out their regex additions. Can you please run that and see if the escaping makes the matches occur properly? Thanks.

Reply:
Looks great @doctoryes.

Consider adding something like:
```
'Example message that should not match'.match(x)
```

Still has 👍 from me.

Reply:
@doctoryes: Also, for the PR where you are setting this, consider a comment in yml with:
1. Link to instructions, 
2. Example error messages that match each section of the regex.

Reply:
I added the non-matching example. And I'll add a link to the instructions here to the YML after this PR merges.